content/blog/2023-02-28-ethereum-classic-is-a-commodity-ethereum-is-a-security/index.md

```diff
@@ -0,0 +1,140 @@
+
+The PoS system is not a free merit system as PoW but a totally collaborative partnership between the pools and the validators, and the validators are essentially the same as the pools because they are their submissive contractors.
+
+In the system, validators divide their tasks on a per block basis where one produces a block, then sends it to the other validators who vote on the block, and then send it to the the rest of the network who must accept them without objection.
+
+Because ETC is PoW and decentralized, miners may pop up or leave whenever they want from anywhere in the world, compete to build blocks in their complete isolation, send them to the rest of the network for verification, and get paid based on their sole merit. No other filters or conditions exist. 
+
+In ETC, the same reasons why it was a commodity before remains today: There is no common enterprise, therefore the future value of the token is just determined by future general adoption by the market, but not from a defined and collaborative group of centralized pool operators or managers as in Ethereum.
+
+## A Historical Perspective of ETC’s and ETH’s Legal Status
+
+![The crowdsale.](./2.png)
+
+From 2014 to 2016 both Ethereum and Ethereum Classic were one project, so we refer to them in this period as ETH/ETC. 
+
+The project was financed with a crowdsale. Based on the Howey test it may be inferred that the ETH/ETC [crowdsale was, indeed, a security](https://twitter.com/WendellMoone/status/1623760982672564226). 
+
+It was [founded](https://en.wikipedia.org/wiki/Ethereum) by Vitalik Buterin; then he partnered with several individuals to form a group of co-founders; they promised the expectation of profits out of the investment; created a foundation with the name of the network, even registering its trademarks; sold a unit of value previous to the creation of the cryptocurrency to gather funds for development; and promoted the crowdsale and Ethereum’s features, benefits, and appreciation potential just like any initial public offering of a stock would have been promoted.
+
+However, this status as a security was likely limited to the period between the crowdsale and the launch of the network on July 30th of 2015. 
+
+![ETH/ETC launch.](./3.png)
+
+On July 30th of 2015, the initial securities of the project were transformed into the ETH cryptocurrency, which was subsequently categorized as a commodity by the CFTC.
+
+This step may be interpreted to be an exchange and exit of the initial common enterprise between investors, Vitalik Buterin, his partners, and the Ethereum Foundation as the director of the project.
+
+From here, the project became a truly decentralized, proof of work based, public blockchain. 
+
+![The Ethereum split.](./7.png)
+
+Even when Ethereum split from the ETC mainnet in 2016 due to the DAO hard fork, ETH kept its decentralization because it remained as a proof of work blockchain.
```

Comment:
Line 102: Usage; Capitalization
```suggestion
Even after Ethereum split from the ETC mainnet in 2016 due to the TheDAO hard fork, ETH kept its decentralization because it remained a proof of work blockchain.
```